testbed/tests/perf_test.go

```diff
@@ -188,3 +158,81 @@ func Test1000SPSWithAttributes(t *testing.T) {
 		})
 	}
 }
+
+
+func Test1000SPSWithAttributes(t *testing.T) {
```

Comment:
This uses the same expectedMaxCPU as the one without ballast. However test results that you posted show that is actually uses less CPU. So, perhaps tighten a bit expectedMaxCPU in this test? I suggest you leave about 30% of slack in CPU (that was my rule of thumb when writing other perf tests).

Reply:
I think I did reduce the numbers before creating the PR. Probably I missed something in the first push. In any case, updated the numbers and reduced them a further.